code/modules/client/client_saveSQL.dm

```diff
@@ -0,0 +1,752 @@
+//handles saving to SQL Format
+//MAKE SURE YOU KEEP THIS UP TO DATE!
+//if the db can't be updated, return 0
+//if the db was updated, return 1
+var/chardata[]
+var/jobsdata[]
+var/flavourdata[]
+var/miscdata[]
+var/DBQuery/query
+var/TextQuery
+var/char_id
+var/char_slot
```

Comment:
As said, I'm fairly certain these are considered global variable definitions. Which manes they'll be overwritten the moment another player loads a new character.

It might be an idea to attach them to the preferences datum, in code\modules\client\preferences.dm?